index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+    <head>
+
+    </head>
+    <body>
+        <h1>Hello World!</h1>
+    </body>
+    <div class="container">
+        <header class="header">
+          <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+          <p id="description" class="description text-center">
+            Thank you for taking the time to help us improve the platform
+          </p>
+        </header>
```

Comment:
Please insert one after closing each tag. This will improve readability of you code.

Reply:
Kindly be specific and more explicit in what you want the students to do wink , this will help him understand the request changes :upside_down_face: , as required in the previous review

Reply:
- [ OPTIONAL] @Javed-Shahbaz good job so far to make some effort but you are not allowed to request the student to use Microverse readme template this should automatically invalid then you explain the reason you invalidate the project then the student can make changes according to the reason why you invalidate his project, next time make sure you take care of this :wink: